local-dev-first-start-only.sh

```diff
@@ -426,35 +371,54 @@ echo '##########################################################################
 # TODO - fix hydroshare container permissions to allow use of hydro-service user
 echo
 echo " - docker exec solr bin/solr create_core -c collection1 -n basic_config"
-docker $DOCKER_PARAM exec solr bin/solr create -c collection1 -d basic_configs
+docker exec solr bin/solr create -c collection1 -d basic_configs
 
 echo
 echo "  - docker exec hydroshare python manage.py build_solr_schema -f schema.xml"
 echo
-docker $DOCKER_PARAM exec hydroshare python manage.py build_solr_schema -f schema.xml
+docker exec hydroshare python manage.py build_solr_schema -f schema.xml
 
 echo
 echo "  - docker cp schema.xml solr:/opt/solr/server/solr/collection1/conf/schema.xml"
 echo
-docker $DOCKER_PARAM cp schema.xml solr:/opt/solr/server/solr/collection1/conf/schema.xml
+docker cp schema.xml solr:/opt/solr/server/solr/collection1/conf/schema.xml
 
 echo
 echo "  - docker exec solr sed -i '/<schemaFactory class=\"ManagedIndexSchemaFactory\">/,+4d' /opt/solr/server/solr/collection1/conf/solrconfig.xml"
-docker $DOCKER_PARAM exec solr sed -i '/<schemaFactory class="ManagedIndexSchemaFactory">/,+4d' /opt/solr/server/solr/collection1/conf/solrconfig.xml
+docker exec solr sed -i '/<schemaFactory class="ManagedIndexSchemaFactory">/,+4d' /opt/solr/server/solr/collection1/conf/solrconfig.xml
 
 echo
 echo "  - docker exec solr rm /opt/solr/server/solr/collection1/conf/managed-schema"
-docker $DOCKER_PARAM exec solr rm /opt/solr/server/solr/collection1/conf/managed-schema
+docker exec solr rm /opt/solr/server/solr/collection1/conf/managed-schema
 
 echo '  - docker exec -u hydro-service hydroshare curl "solr:8983/solr/admin/cores?action=RELOAD&core=collection1"'
 echo
-docker $DOCKER_PARAM exec -u hydro-service hydroshare curl "solr:8983/solr/admin/cores?action=RELOAD&core=collection1"
+docker exec -u hydro-service hydroshare curl "solr:8983/solr/admin/cores?action=RELOAD&core=collection1"
+
+echo
+echo '########################################################################################################################'
+echo " Collect static files"
+echo '########################################################################################################################'
+
+# check to see if the node container is still running
+# if it is, wait until it is removed
+echo "Waiting for nodejs container to be removed..."
+while [ 1 -eq 1 ]
+do
+  if [ "$NODE_CONTAINER_RUNNING" == "" ]; then
+    break
+  fi
+  echo -n "."
+  sleep 1
+done
 
-docker-compose -f local-dev.yml down
```

Comment:
ahh, I see that you leave HS running now!

Reply:
yeah I thought it would save some time